.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,43 @@
-<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
-v                               ✰  Thanks for creating a PR! ✰    
-v    Before smashing the submit button please review the checkboxes.
-v    If a checkbox is n/a - please still include it but + a little note why
-☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
-
 ## Description
-<!-- Small description -->
-
-## Checklist
-- [ ] Targeted PR against correct branch.
-- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
-- [ ] Wrote unit tests.
-- [ ] Wrote integration tests (simulation & CLI). 
-- [ ] Updated the documentation. 
-- [ ] Added an entry to the `CHANGELOG.md` file.
-- [ ] Re-reviewed `Files changed` in the Github PR explorer.
+
+Closes: #XXXX
+
+<!-- Add a description of the changes that this PR introduces and the files that
+are the most critical to review. -->
+
+---
+
+### Author Checklist
+
+*All items are required. Please add a note to the item if the item is not applicable and
+please add links to any relevant follow up issues.*
+
+I have...
+
+- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
+- [ ] added `!` to the type prefix if API or client breaking change
+- [ ] targeted the correct branch (see [PR Targeting](https://github.com/desmos-labs/desmos/blob/master/CONTRIBUTING.md#pr-targeting))
+- [ ] provided a link to the relevant issue or specification
+- [ ] followed the guidelines for [building modules](https://github.com/desmos-labs/desmos/blob/master/docs/building-modules)
```

Comment:
The link of building modules is not found.

Reply:
Fixed